test/unittests/t_options.cc

```diff
@@ -0,0 +1,110 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+
+#include "gtest/gtest.h"
+
+#include "../../cvmfs/options.h"
+#include "../../cvmfs/util.h"
+
+using namespace std;  // NOLINT
+
+template <class OptionsT>
+class T_Options : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    FILE *temp_file = CreateTempFile("/tmp/cvmfs-test", 0600, "w",
+        &config_file_);
+    ASSERT_TRUE(temp_file != NULL);
+    unlink_guard_.Set(config_file_);
+    fprintf(temp_file,
```

Comment:
The indentation in this `fprintf` statement looks still strange.  Can you check?